test/functional/outputs/cloudwatch/forward_to_cloudwatch_test.go

```diff
@@ -144,6 +145,43 @@ var _ = Describe("[Functional][Outputs][CloudWatch] Forward Output to CloudWatch
 
 		})
 
+		It("should be able to forward by user-defined pod labels", func() {
+			var (
+				labelKey   = "env1"
+				labelValue = "test1"
+			)
+			// Add our new label
+			framework.Labels[labelKey] = labelValue
+
+			// Generate a pod spec with a selector for labels
+			functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+				FromInputWithVisitor("app-logs",
+					func(spec *logging.InputSpec) {
+						spec.Application = &logging.Application{
```

Comment:
suggest to add a Namespace name also here. this is how Application Input is usually defined, for a specific namespace. otherwise lgtm